.github/workflows/shared-build/action.yml

```diff
@@ -59,7 +59,7 @@ runs:
     - run: npm dedupe
       working-directory: testproject
       shell: bash
-    - run: pip install poetry==1.7.1 --upgrade
+    - run: pip install poetry==1.8.3 --upgrade
```

Comment:
https://github.com/dependabot/dependabot-core/issues/9209

I needed to update poetry.